backend/apps/issue_quality/services/quality_scorer.py

```
@@ -0,0 +1,404 @@
+            
+            result = {
+                'language': language,
+                'is_english': is_english,
+                'translation': ''
+            }
+            
+            if not is_english:
+                try:
+                    translation = self.translator.translate(text, dest='en')
+                    result['translation'] = translation.text
+                except:
```

Ruff (0.15.21)

[error] Do not use bare `except`

(E722)